serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -57,7 +57,7 @@
 ** xref:cloud/deploying-on-minikube.adoc[Deploying your {context} application on Minikube]
 // ** xref:cloud/deploying-on-kubernetes-cluster.adoc[Deploying on Kubernetes Clusters]
 // ** xref:cloud/versioning-workflows-in-knative.adoc[Versioning workflows in Knative]
-** xref:cloud/kubernetes-service-discovery.adoc[Kubernetes Service Discovery for Serverless Workflow Application]
+** xref:cloud/kubernetes-service-discovery.adoc[Kubernetes service discovery in {context}]
```

Comment:
not sure what context will be translated to, wouldn't be better just use Kubernetes service discovery?

Reply:
the {context} will be translated to Serverless Workflow in upstream and OpenShift Serverless logic in downstream.

Reply:
okay, thanks.